pegasus/sites.v3/code.org/views/carousel_computer_works_video.haml

```diff
@@ -0,0 +1,28 @@
+- video_computer_index = 6
+- video_computer_id = ["computer_works", "what_computer", "binary_data", "circuits_logic", "cpu_memory", "hardware_software"]
+- video_computer_caption = [hoc_s(:video_library_page_computer_work_video_title_1), hoc_s(:video_library_page_computer_work_video_title_4), hoc_s(:video_library_page_computer_work_video_title_2),
+ hoc_s(:video_library_page_computer_work_video_title_5), hoc_s(:video_library_page_computer_work_video_title_3), hoc_s(:video_library_page_computer_work_video_title_6)]
+- video_computer = ["OAx_6-wdslM", "mCq8-xTH7jA", "USCBCmwMCDA", "ZoqMiFKspAA", "DKGZlaPlVLY", "xnyFYiK2rSY"]
+
+
+.carousel-wrapper#computer_works_collection
+  .carousel.computer__works__collection
+    .slides
+      - video_computer_index.times do |index|
+        .slide
+          .action-block.action-block--one-col.flex-space-between
+            .video__container
+              = view :display_video_thumbnail, id: video_computer_id[index], video_code: video_computer[index], play_button: 'center', letterbox: 'false', caption: video_computer_caption[index]
+
+            
+  .pagination
+    #computer_works_collection_pagination
+  .dd__buttons
+    %button.carousel__prev#prev_3
+      %img{src: "/images/carousel/prev.svg", alt: "previous"}
+
+    %button.carousel__next#next_3
+      %img{src: "/images/carousel/next.svg", alt: "next"}
+
+
+
```

Comment:
Nit: extra whitespace — there can just be one line of whitespace at the end of the file